pangeo_forge_recipes/writers.py

```diff
@@ -95,22 +97,44 @@ def store_dataset_fragment(
 def write_combined_reference(
     reference: MultiZarrToZarr,
     full_target: FSSpecTarget,
-    output_json_fname: str,
+    concat_dims: List[str],
+    output_file_name: str,
+    output_file_type: str,
+    refs_per_component: int = 1000,
 ):
     """Write a kerchunk combined references object to file."""
 
     import ujson  # type: ignore
 
-    multi_kerchunk = reference.translate()
-    file_ext = os.path.splitext(output_json_fname)[-1]
-
-    if file_ext == ".json":
-        outpath = os.path.join(full_target.root_path, output_json_fname)
+    outpath = os.path.join(full_target.root_path, output_file_name + "." + output_file_type)
+    if output_file_type == "json":
+        multi_kerchunk = reference.translate()
         with full_target.fs.open(outpath, "wb") as f:
             f.write(ujson.dumps(multi_kerchunk).encode())
+
+    elif output_file_type == "parquet":
+
+        # Creates empty parquet store to be written to
+        if os.path.exists(outpath):
+            import shutil
+
+            shutil.rmtree(outpath)
+        os.makedirs(outpath)
```

Comment:
This will presumably fail against cloud storage.

Rather than `shutil` and `os`, we should be using the cloud-storage-aware methods on the `full_target` kw passed to this function. Available methods on the class definition here:


https://github.com/pangeo-forge/pangeo-forge-recipes/blob/356ea2a64e7e15dff27c2497f232a4312b2debc7/pangeo_forge_recipes/storage.py#L85

Let's try to make it work wit the existing methods there, but if not we could add methods as needed.